Ginger/GingerCore/GeneralLib/General.cs

```diff
@@ -431,20 +441,37 @@ public static bool IsNameAlreadyexists(RepositoryItemBase repositoryItem, string
                     }
 
                     break;
-                case "EnvApplication":
-                    if (WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<ProjEnvironment>().Any(x => x.Applications.Any(y=>y.Name== resultValue)))
+                case "Environment":
+                    if (WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<ProjEnvironment>().Any(x => x.Name == resultValue))
                     {
-                        Reporter.ToUser(eUserMsgKey.StaticWarnMessage, "Application with same name: " + "'" + resultValue + "'" + " already exists.");
+                        Reporter.ToUser(eUserMsgKey.StaticWarnMessage, "Environment with same name: " + "'" + resultValue + "'" + " already exists.");
```

Comment:
Use string interpolation to concat string

Reply:
Done